src/EnergyPlus/PackagedThermalStorageCoil.cc

```diff
@@ -2318,23 +2310,23 @@ void InitTESCoil(EnergyPlusData &state, int &TESCoilNum)
         state.dataPackagedThermalStorageCoil->TESCoil(TESCoilNum).ElectEvapCondBasinHeaterPower = 0.0;
         state.dataPackagedThermalStorageCoil->TESCoil(TESCoilNum).ElectEvapCondBasinHeaterEnergy = 0.0;
 
-        MyEnvrnFlag(TESCoilNum) = false;
+        state.dataPackagedThermalStorageCoil->MyEnvrnFlag(TESCoilNum) = false;
     }
 
-    if (!state.dataGlobal->BeginEnvrnFlag) MyEnvrnFlag(TESCoilNum) = true;
+    if (!state.dataGlobal->BeginEnvrnFlag) state.dataPackagedThermalStorageCoil->MyEnvrnFlag(TESCoilNum) = true;
 
-    if (MyWarmupFlag(TESCoilNum) && (!state.dataGlobal->WarmupFlag)) {
+    if (state.dataPackagedThermalStorageCoil->MyWarmupFlag(TESCoilNum) && (!state.dataGlobal->WarmupFlag)) {
         // reset to initial condition once warm up is over.
         state.dataPackagedThermalStorageCoil->TESCoil(TESCoilNum).IceFracRemain = 0.0;
```

Comment:
`state.dataPackagedThermalStorageCoil->TESCoil(TESCoilNum)` can use a shortcut?

Reply:
Same comment. Different PR. Probably one that will do a general cleanup on this module.

Reply:
same here